pegasus/sites.v3/code.org/views/top_hoc_tutorials.haml

```diff
@@ -1,7 +1,22 @@
 = view :top_hoc_tutorial_applab, image_match_youtube_height: false
 
+- outbreak_image_url = "/images/outbreak/outbreak.png"
 - minecraft_img_url = CDO.code_org_url("/shared/images/fill-485x260/courses/logo_tall_aquatic.jpg")
 
+.col-50
+  .tutorial-tile
+    %a{href: CDO.studio_url("/s/outbreak/lessons/1/levels/1")}
+      %img.tutorial-tile-img{src: CDO.code_org_url("/images/outbreak/outbreak.png")}
```

Comment:
Could you add `alt` text for the image so we can support a11y tools? You will probably need another i18n string which describes the image.